scripts/mbedtls_framework/code_wrapper/psa_test_wrapper.py

```diff
@@ -31,7 +31,7 @@ class PSATestWrapper(PSAWrapper):
     _WRAPPER_NAME_PREFIX = 'mbedtls_test_wrap_'
     _WRAPPER_NAME_SUFFIX = ''
 
-    _PSA_WRAPPER_INCLUDES = ['<psa/crypto.h>',
+    _PSA_WRAPPER_INCLUDES = ['<psa/crypto.h>\n',
```

Comment:
This newline is not required but alligns the generated files to what is already comitted

    #include <psa/crypto.h>

    #include <test/memory.h>
    #include <test/psa_crypto_helpers.h>
    #include <test/psa_test_wrappers.h>

Reply:
I'd prefer to have an unimportant newline disappear here than have this bizarre newline in an include directive argument.